src/DIRAC/FrameworkSystem/Utilities/diracx.py

```diff
@@ -27,6 +27,8 @@
 DEFAULT_TOKEN_CACHE_SIZE = 1024
 
 legacy_exchange_session = requests.Session()
+diracxUrl = gConfig.getValue("/DiracX/URL")
```

Comment:
This should be a URL, or you break the pydantic check.

Probably:

```suggestion
diracxUrl = gConfig.getValue("/DiracX/URL", os.environ["DIRACX_URL"])
```
then inject one for the unit test.